lib/services/tokens/token.service.js

```diff
@@ -160,7 +160,8 @@ s.createJWT = function (payload) {
       issuer: config.systemConfig.accessTokens.issuer,
       audience: config.systemConfig.accessTokens.audience,
       expiresIn: config.systemConfig.accessTokens.timeToExpiry,
-      subject: config.systemConfig.accessTokens.subject
+      subject: config.systemConfig.accessTokens.subject,
+      algorithm: config.systemConfig.accessTokens.algorithm || "HS256"
```

Comment:
We do not put default values in the code as long it is possible. Please modify the relative [json schema](https://github.com/ExpressGateway/express-gateway/blob/master/lib/config/schemas/system.config.json#L55) where you can put all the choices and the default value.